variables.tf

```
@@ -22,6 +22,7 @@ variable "project_id" {
 variable "job_name" {
   type        = string
   description = "The name of the scheduled job to run"
+  default     = ""
```

We should use `null` rather than an empty string.

```suggestion
  default     = null
```